### PR TITLE
fix: typos and minor UI project issues

### PR DESCRIPTION
**Pull request by evanshortiss**

### CODE REVIEW

File: docs/labs/01-Event-Driven-Architecture/walkthrough.adoc [hunk 1]
```diff
@@ -1551,8 +1551,10 @@ app.post('/order/event', json(), async (req, res, next) => {
   log.info('placing order with body: %j', req.body)
   try {
     const response = await http.post(new URL('/place', ORDERS_EVENT_BASE_URL ? ORDERS_EVENT_BASE_URL : "http://i-events.fuse-{user-username}.svc:8080"), {
-      json: true,
-      body: req.body
+      headers: {
```
Comment (evanshortiss) [inline docs/labs/01-Event-Driven-Architecture/walkthrough.adoc:1554]: The Fuse Online service does not return JSON so setting `json: true` causes an error when the Node.js application attempts to `JSON.parse` the response.

File: projects/store-ui/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "build": "npm run copy:patternfly-assets",
     "copy:patternfly-assets": "cp -R node_modules/@patternfly/patternfly/assets public && cp node_modules/@patternfly/patternfly/patternfly.min.css public/patternfly.min.css && cp node_modules/@patternfly/patternfly/patternfly-addons.css public/patternfly-addons.css",
     "coverage": "nyc npm test",
-    "openshift": "nodeshift --strictSSL=false --nodeVersion=10.x --expose --quiet",
+    "openshift": "npx nodeshift --strictSSL=false --nodeVersion=10.x --expose --quiet",
```
Comment (evanshortiss) [inline projects/store-ui/package.json:12]: Simplifies the command needed in the adoc file.

File: docs/labs/01-Event-Driven-Architecture/walkthrough.adoc [hunk 3]
```diff
@@ -524,7 +524,7 @@ For the following commands, remember to be sure to switch to your _working_ proj
 +
 [source,bash,subs="attributes+"]
 ----
-npm install --save-dev nodeshift; npm run openshift
+npm run openshift
```
Comment (evanshortiss) [inline docs/labs/01-Event-Driven-Architecture/walkthrough.adoc:527]: Linked to the change in package.json

**Comment by hguerrero** [comment]
@evanshortiss thanks for the collaboration. We are working on the new instructions on the `module-split` branch. Can you move this changes there?

**Comment by evanshortiss** [comment]
@hguerrero, sure!

**Pull request closed without merging**